src/components/TransactionConfirmationModal/index.tsx

```diff
@@ -63,15 +64,34 @@ export function ConfirmationPendingContent({ onDismiss, pendingText }: { onDismi
 }
 
 function TransactionSubmittedContent({
+  trade,
   onDismiss,
   chainId,
   hash,
 }: {
+  trade?: Trade
   onDismiss: () => void
   hash: string | undefined
   chainId: ChainId
 }) {
   const theme = useTheme()
+  const { t } = useTranslation()
+
+  const isGnosisProtocolTrade = trade instanceof GnosisProtocolTrade
+  const explorer = isGnosisProtocolTrade ? t('gnosisProtocolExplorer') : t('blockExplorer')
+  const explorerExternalLink = chainId && hash && (
+    <ExternalLink
+      href={
+        isGnosisProtocolTrade
```

Comment:
same, I thinks it's cleaner to have a variable link which return the correct link like in here, https://github.com/levelkdev/swapr-dapp/pull/1009/files#diff-6bf8d2c7365c6b92df4d531ac3cef0e16595df3d8e0e34ba6eb17d1a6c7b63b7R51

Reply:
Done.